src/preprocessing/spacy_preprocess.py

```diff
@@ -1,26 +1,29 @@
 # Author: Jenit Jain
 # Date: 2023-06-21
-"""This script manages the custom training and fine tuning of spacy models.
+"""This script manages the custom data artifact creation for training and fine tuning of spacy models.
 
-Usage: spacy_preprocess.py --data_path=<data_path> --train_split=<train_split> --val_split=<val_split> --test_split=<test_split>
+Usage: spacy_preprocess.py --data_path=<data_path>
 
 Options:
     --data_path=<data_path>         The path to the dataset in JSONLines format.
-    --train_split=<train_split>     The ratio of files to include in the training set.
-    --val_split=<val_split>         The ratio of files to include in the validation set.
-    --test_split=<test_split>       The ratio of files to include in the testing set.
 """
 
 import os
+import sys
 import spacy
 import json
 import logging
 import glob
 from docopt import docopt
 from spacy.tokens import DocBin
 
-logging.basicConfig(level=logging.INFO)
-logger = logging.getLogger(__name__)
+# ensure that the parent directory is on the path for relative imports
+sys.path.append(os.path.join(os.path.dirname(__file__), os.pardir, os.pardir))
+
+from src.logs import get_logger
+logger = get_logger(__name__)
+logger.setLevel(logging.INFO)
```

Comment:
issue: log level should not be set here, the log level should be manage only in the logs.py file, should delete this line

This is because then it's a single place to update the log level for the whole repo, if it's important to be set to Info for this script then when running it set the environment variable `LOG_LEVEL` to `INFO` and the logging will be updated in the logs.py script.